src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -223,9 +223,20 @@ extension RiskProvider: RiskProviding {
 		}
 
 		completeOnTargetQueue(risk: risk)
+
+		saveRiskIfNeeded(risk)
```

Comment:
@ChristianKienle could you please have a look and confirm, that this is correct place for storing previous risk level. thanks

Reply:
yeah looking at it :)